R/format_ay.R

```diff
@@ -17,3 +17,24 @@ format_ay <- function(year) {
   }
   sub("(.{4})(.*)", "\\1/\\2", year)
 }
+
+#' Undo academic year formatting
+#'
+#' This function converts academic year variables back into 201617 format.
+#'
+#' It accepts character arguments.
+#'
+#' @param year Academic year
+#' @return Unformatted year
```

Comment:
Should we specify it returns a character string? 

Second question - is that right, would we want to return a numeric value? (I'm not sure off the top of my head which way is best so interested in your thoughts)

Reply:
Hmm.... not sure. I guess it's easier to convert from character > date so maybe leave it as character? I had to make the input a character variable otherwise it starts trying to do sums with the numbers

Reply:
That's fair, in that case I think we update the @return to something like 'Unformatted 6 digit year as string'?